### DISCUSSION TITLE
Error: Reached the maximum number of users

### DISCUSSION THREAD

**Discussion by ertce37** [etsy/open-api#1268] [Q&A] [answered]
Hi, 
When I try to authenticate new shops into my app, I start seeing this Error: Reached the maximum number of users. Does anyone know why started getting this error? Thanks in advance. 
![etsy error message](https://github.com/user-attachments/assets/2eda9d16-8cf4-46ce-a1c7-4d30183859be)

**Comment by turboturtleD** [comment, marked as answer]
It’s a pretty straightforward error.  You need to request commercial status for your app if you want to authenticate multiple shops.  You may be able to request an additional 1 or 2 users so you can auth all your shops (but probably not), otherwise you would need to have and use credentials for each shop.  You may find that concept to have pitfalls as well as they want to make sure that you aren’t having other shops hand you their credentials or plugging them directly into your app.

If you are hoping to have other shops which do not belong to you to authorize your app, you will need to request commercial access.

**Comment by Wilderbill** [comment]
See:  https://github.com/slider1234/ETSY-API-Auto-Authenticator

I got various error messages during the OAuth process (including the max number of users).

The cause of my problems was multiple overlapping instances as revealed by "Apps You Use".
( See the website: https://www.etsy.com/your/apps?ref=si_apps )
I had a half dozen active instances of my app as a result of my flailing attempts to get it working.
Apparently, there can be multiple instances of your app with overlapping, conflicting scopes.
The solution was to delete all of my instances and start again.
Set the scopes you want and run: https://github.com/slider1234/ETSY-API-Auto-Authenticator again.
( This does not require a repeat of the 3-week application process. )
